ci-operator/config/konveyor/mtrho-operator/konveyor-mtrho-operator-main.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
+  ubi8-minimal:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-base-openshift-4.11
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile
+  from: ubi8-minimal
+  to: mtrho-operator
+operator:
+  bundles:
+  - dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/konveyor/mtrho-operator-container:latest
+    with: mtrho-operator
+promotion:
+  name: "4.11"
+  namespace: ocp
```

Comment:
I made this `konveyor` (in my not yet submitted PR) as I think that may make it easier to keep all of our testable images together. wdyt?

Reply:
once the current round of tests finishes I'll update it.

Reply:
In the end I'm not sure it's worth it because then I think we have to figure out the image mapping on our own.

Reply:
ACK. If it will be helpful in the future and we can make it work, we can always change it.